### ISSUE TITLE
Minifying problem with create-react-app build

### ISSUE THREAD

**Issue by vietnugent** [rsms/js-lru#30]
Thanks for this really useful library!  

My project was created from FB `create-react-app`.  When trying to prepare the production bundle the build step fails due to minifying issue with js-lru.   I'm quoting their explanation for the error:

>Some third-party packages don't compile their code to ES5 before publishing to npm. This often causes problems in the ecosystem because neither browsers (except for most modern versions) nor some tools currently support all ES6 features. We recommend to publish code on npm as ES5 at least for a few more years.

https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#npm-run-build-fails-to-minify

```
# npm run build
...
Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file:

        ./node_modules/lru_map/lru.js:85

Read more here: http://bit.ly/2tRViJ9
```

**Comment by roboslone** [comment]
Same here with `webpack v3.8.1`:
```
ERROR in yappy.min.js from UglifyJs
Unexpected token: name (entry) [./node_modules/lru_map/lru.js:85,0][yappy.min.js:111219,6]
```

Is there a workaround?

**Comment by rsms** [comment]
Fixed in 2bd72701f507d494d08a14bf4fbfbabd21c602ba
